src/include/btree.i

```diff
@@ -261,7 +262,7 @@ __wt_cache_page_byte_dirty_decr(
 		    decr, "WT_BTREE.bytes_dirty_intl");
 		__wt_cache_decr_check_uint64(session, &cache->bytes_dirty_intl,
 		    decr, "WT_CACHE.bytes_dirty_intl");
-	} else if (!btree->lsm_primary) {
+	} else if (!btree->lsm_primary && !F_ISSET(btree, WT_BTREE_LOOKASIDE)) {
```

Comment:
This needs careful thought.  With this change, lookaside pages no longer contribute to the "dirty bytes in cache".  My thinking:

1. lookaside isn't involved in checkpoints, so from that point of view doesn't contribute to the work checkpoints have to do; and
2. evicting a (dirty) page by writing entries to lookaside should contribute to unblocking operations, rather than requiring a second eviction of the lookaside page(s) before the dirty bytes in cache goes down.

That said, this goes against some of the purpose of the 20% limit on dirty bytes in cache, which was to limit the impact of memory allocation fragmentation from lots of small allocations.